sympy/polys/orthopolys.py

```diff
@@ -37,8 +37,10 @@ def dup_jacobi(n, a, b, K):
 
 
 @public
-def jacobi_poly(n, a, b, x=None, **args):
-    """Generates Jacobi polynomial of degree `n` in `x`. """
+def jacobi_poly(n, a, b, x=None, polys=False):
+    """Generates Jacobi polynomial of degree `n` in `x`.
+    Returns a Poly object when polys=True, otherwise
```

Comment:
we can write in brackets `(default)` after "otherwise". like
```
otherwise (default) returns an expression
```

Also use \`\`polys=True\`\` (i.e double backtick it).